web/elm/src/SideBar/Styles.elm

```diff
@@ -141,43 +141,55 @@ opacityAttr opacity =
                 "1"
 
 
-teamIcon : Opacity -> Html.Html msg
-teamIcon opacity =
+colorAttr : Opacity -> Html.Attribute msg
+colorAttr opacity =
+    style "color" <|
+        case opacity of
+            Dim ->
+                Colors.sideBarTextDim
+
+            GreyedOut ->
+                Colors.sideBarTextDim
+
+            Bright ->
+                Colors.sideBarTextBright
+
+
+teamIcon : Html.Html msg
+teamIcon =
     Icon.icon
         { sizePx = 18
         , image = Assets.PeopleIcon
         }
         [ style "margin-left" "8px"
         , style "background-size" "contain"
         , style "flex-shrink" "0"
-        , opacityAttr opacity
         ]
 
 
 collapseIcon : { opacity : Opacity, asset : Assets.Asset } -> Html.Html msg
-collapseIcon { opacity, asset } =
+collapseIcon { asset } =
     Icon.icon
         { sizePx = 10
         , image = asset
         }
         [ style "margin-left" "10px"
         , style "flex-shrink" "0"
-        , opacityAttr opacity
         ]
 
 
 teamName :
     { a | opacity : Opacity }
     -> List (Html.Attribute msg)
-teamName { opacity } =
+teamName _ =
     [ style "font-size" "14px"
     , style "padding" "5px 2.5px"
     , style "margin-left" "5px"
     , style "white-space" "nowrap"
     , style "overflow" "hidden"
     , style "text-overflow" "ellipsis"
     , style "flex-grow" "1"
-    , opacityAttr opacity
+    , style "color" Colors.sideBarTextBright
```

Comment:
Is there a reason we want this, rather than `colorAttr`?

The current behaviour on master is to highlight the currently selected team in the sidebar:

<img width="360" alt="Screen Shot 2020-11-03 at 10 20 37 AM" src="https://user-images.githubusercontent.com/26583442/98004744-48f5eb80-1dbe-11eb-8eac-a1f30f92ca41.png">

which makes it easy to see what team that the open pipeline/job/resource belongs to. With this, all teams appear the same:

<img width="413" alt="Screen Shot 2020-11-03 at 10 13 07 AM" src="https://user-images.githubusercontent.com/26583442/98004939-83f81f00-1dbe-11eb-8690-cfcb69835970.png">

Reply:
i think this was just an oversight on my part! i'm going to go with the values matt specified here: https://github.com/concourse/concourse/pull/6168#discussion_r516796395